managementApp/src/main/java/com/skytask/controller/ProductController.java

```diff
@@ -0,0 +1,42 @@
+package com.skytask.controller;
+
+import com.skytask.channel.ProductSource;
+import com.skytask.common.Product;
+import com.skytask.service.ProductService;
+import org.springframework.cloud.sleuth.Tracer;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+class ProductController {
+
+    private ProductService productService;
+    private ProductSource productSource;
+    private Tracer tracer;
+
+    public ProductController(ProductService productService, ProductSource productSource, Tracer tracer) {
+        this.productService = productService;
+        this.productSource = productSource;
+        this.tracer = tracer;
+    }
+
+    @RequestMapping(value = "/", method = RequestMethod.GET)
+    public ModelAndView list() {
+        productSource.getProductList().send(MessageBuilder.withPayload("getProductsList").setCorrelationId(tracer.getCurrentSpan().getTraceId()).build());
+        return new ModelAndView("index", "products", productService.getProducts());
+    }
+
+    @RequestMapping(value = "/create", method = RequestMethod.GET)
```

Comment:
by standard REST paths should name resources and not actions, the action it does it's supposed to be represented by the HTTP verb (GET, POST) for example GET "/product" retrieves a product, POST "/product" creates a product, and so on.

Reply:
It is a get for a view form in order to create a product, would something like /createform be ok? Or what name could I put there?

Reply:
ok, so you're not using it as a rest controller, nvm 👍